conf/lookup/BCD-Port.csv

```diff
@@ -15520,4 +15518,101 @@ PortId,CountryCode,PortType,PortCode,KeyWords,Name,Name-cy,Priority,Region,Locat
+200027,UK,3,FOL,,Rye (Folkestone) - FOL,Rye (Folkestone) - FOL,,1,NULL,0,
+200028,UK,3,FOY,Charlestown,Fowey - FOY,Fowey - FOY,,1,NULL,0,
+200029,UK,3,FRB,Buckie Macduff,Fraserburgh - FRB,Fraserburgh - FRB,,1,NULL,0,
+200030,UK,3,GOO,Howdendyke Selby,Goole - GOO,Goole - GOO,,1,NULL,0,
+200031,UK,3,GRG,Braefoot Bay Burntisland Inverkeithing Methil,Grangemouth - GRG,Grangemouth - GRG,,1,NULL,0,
+200032,UK,3,GRI,,Grimsby - GRI,Grimsby - GRI,,1,NULL,0,
+200033,UK,3,GRK,Annan Barcaldine Campbeltown Faslane FinnartFurnace Garlieston Hunterston Islay Oban Palnackie Stranraer,Greenock - GRK,Greenock - GRK,,1,NULL,0,
+200034,UK,3,GSY,,St Peter Port (Guernsey) - GSY,St Peter Port (Guernsey) - GSY,,1,NULL,0,
+200035,UK,3,GTY,,Great Yarmouth - GTY,Great Yarmouth - GTY,,1,NULL,0,
+200036,UK,3,HEY,Lancaster,Heysham - HEY,Heysham - HEY,,1,NULL,0,
+200037,UK,3,HLD,,Holyhead - HLD,Holyhead - HLD,,1,NULL,0,
+200038,UK,3,HPT,,Hound Point TerminalHPT,Hound Point TerminalHPT,,1,NULL,0,
+200039,UK,3,HRH,Mistley Parkeston,Harwich International Port - HRH,Harwich International Port - HRH,,1,NULL,0,
+200040,UK,3,HTP,,Hartlepool - HTP,Hartlepool - HTP,,1,NULL,0,
+200041,UK,3,HUL,Bridlington,Hull - HUL,Hull - HUL,,1,NULL,0,
+200042,UK,3,IMM,Barrow-On-Humber Barton-On-Humber New Holland,Immingham - IMM,Immingham - IMM,,1,NULL,0,
+200043,UK,3,INV,Burghead Corpach Incl Annat Pier Lochaline Lossiemouth Scrabster Ullapool Wick ,Inverness Ports - INV,Inverness Ports - INV,,1,NULL,0,
+200044,UK,3,IPS,,Ipswich - IPS,Ipswich - IPS,,1,NULL,0,
+200045,UK,3,IRV,,Irvine - IRV,Irvine - IRV,,1,NULL,0,
+200046,UK,3,IVG,,Invergordon - IVG ,Invergordon - IVG ,,1,NULL,0,
+200047,UK,3,JSY,,St Helier (Jersey) - JSY,St Helier (Jersey) - JSY,,1,NULL,0,
+200048,UK,2,KEM,,Kemble Airport - KEM,Kemble Airport - KEM,,1,NULL,0,
+200049,UK,3,KIL,,Killingholme - KIL,Killingholme - KIL,,1,NULL,0,
+200050,UK,3,KKD,,Kirkcaldy - KKD,Kirkcaldy - KKD,,1,NULL,0,
+200051,UK,3,KLN,,Kings Lynn - KLN,Kings Lynn - KLN,,1,NULL,0,
+200052,UK,3,KWL,Stromness,Kirkwall - KWL,Kirkwall - KWL,,1,NULL,0,
+200053,UK,2,LAS,,Lasham Airport - LAS,Lasham Airport - LAS,,1,NULL,0,
+200054,UK,3,LBK,,Barking - LBK,Barking - LBK,,1,NULL,0,
+200055,UK,3,LEI,,Leith - LEI,Leith - LEI,,1,NULL,0,
+200056,UK,3,LER,Scalloway,Lerwick - LER,Lerwick - LER,,1,NULL,0,
+200057,UK,3,LGP,,London Gateway - LGP,London Gateway - LGP,,1,NULL,0,
+200058,UK,3,LKH,,Lakenheath - LKH,Lakenheath - LKH,,1,NULL,0,
+200059,UK,3,LLA,Caernarfon Port Penrhyn,Llanddulas - LLA,Llanddulas - LLA,,1,NULL,0,
+200060,UK,3,LOS,,Lossiemouth - LOS,Lossiemouth - LOS,,1,NULL,0,
+200061,UK,3,LOW,,Lowestoft - LOW,Lowestoft - LOW,,1,NULL,0,
+200062,UK,2,LYD,,London Ashford (Lydd) Airport - LYD,London Ashford (Lydd) Airport - LYD,,1,NULL,0,
+200063,UK,3,LYH,,Lyneham - LYH,Lyneham - LYH,,1,NULL,0,
+200064,UK,3,MDH,,Mildenhall - MDH,Mildenhall - MDH,,1,NULL,0,
+200065,UK,3,MED,Sheerness Queenborough Ridham Dock Rochester,Medway - MED,Medway - MED,,1,NULL,0,

```

Comment:
Duplicates 3014,UK,3,SHS,,Port of Sheerness Seaport - SHS,Port of Sheerness Seaport - SHS,,1,NULL,1,

Keep both